mdoc.Test.FSharp/DiscriminatedUnions.fs

```diff
@@ -0,0 +1,18 @@
+﻿module DiscriminatedUnions
+
+//The preceding code declares a discriminated union Shape, 
+// which can have values of any of three cases: Rectangle, Circle, and Prism. 
+// Each case has a different set of fields.
+type Shape =
```

Comment:
Upon casual inspection, it seems like a lot of the samples in this section come from the microsoft docs ... for example, this one can be found here:
https://docs.microsoft.com/en-us/dotnet/fsharp/language-reference/discriminated-unions#remarks

Perhaps this is a question for @dend ... but as that content is available on a creative commons/attribution license, do we need to include any kind of attribution in these source files?